db/seeds.rb

```diff
@@ -6,20 +6,54 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-Merchant.destroy_all
-Item.destroy_all
+ItemOrder.destroy_all
```

Comment:
We have lots of seeds. Should we spend time today fixing these and testing our rails server? If they look good we can transfer them to the Heroku database for deployment!

Reply:
Yeah good idea! I'll add a ticket to review seeds file and clean up.